src/textarea.test.jsx

```diff
@@ -93,6 +97,25 @@ describe('Textarea', function() {
 		const boundComponent = TestUtils.renderIntoDocument(<Textarea
 			name={NAME_ATTR}
 			label={LABEL_TEXT}
+			id={ID}
+			value={VALUE} />);
+
+		textareaEl = TestUtils.findRenderedDOMComponentWithTag(boundComponent, 'textarea');
+		TestUtils.Simulate.change(textareaEl, { target: { value: newValue } });
+
+		expect(changeSpy).toHaveBeenCalled();
+		expect(stateSpy).toHaveBeenCalledWith({ value: newValue });
+	});
+	it('should handle onChange as a prop', function() {
+		const newValue = `${VALUE}r`;
+		const changeSpy = spyOn(Textarea.prototype, 'onChange').and.callThrough();
```

Comment:
it looks to be spying on the internal onChange vs the Textarea.props version. You could do something like...
```
		const onChange = jest.fn();
		const boundComponent = TestUtils.renderIntoDocument(
			<Textarea
					onChange={onChange}
					name={NAME_ATTR}
					label={LABEL_TEXT}
					id={ID}
					value={VALUE}
			/>);
			TestUtils.Simulate.change(textareaEl, { target: { value: newValue } });
			expect(onChange).toHaveBeenCalled();
```